packages/ui/DropDown/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import {
+  useFloating,
+  useClick,
+  useInteractions,
+  autoUpdate,
+  offset,
+  flip,
+  shift,
+  useDismiss,
+} from '@floating-ui/react-dom-interactions';
+
+interface DropDownProps {
+  parent: JSX.Element;
+  children: JSX.Element;
+}
+
+export const DropDown = ({ parent, children }: DropDownProps) => {
+  const [open, setOpen] = useState(false);
```

Comment:
pass these as props, to make it reusable.

Reply:
done